iceoryx_posh/source/runtime/posh_runtime.cpp

```diff
@@ -81,18 +82,13 @@ const RuntimeName_t& PoshRuntime::verifyInstanceName(cxx::optional<const Runtime
 {
     if (!name.has_value())
     {
-        LogError() << "Cannot initialize runtime. Application name has not been specified!";
+        LogFatal() << "Cannot initialize runtime. Application name has not been specified!";
         errorHandler(PoshError::POSH__RUNTIME_NO_NAME_PROVIDED, ErrorLevel::FATAL);
     }
-    else if (name.value()->empty())
+    else if (!cxx::isValidFileName(**name))
     {
-        LogError() << "Cannot initialize runtime. Application name must not be empty!";
-        errorHandler(PoshError::POSH__RUNTIME_NAME_EMPTY, ErrorLevel::FATAL);
-    }
-    else if (name.value()->c_str()[0] == '/')
-    {
-        LogError() << "Cannot initialize runtime. Please remove leading slash from Application name " << *name.value();
-        errorHandler(PoshError::POSH__RUNTIME_LEADING_SLASH_PROVIDED, ErrorLevel::FATAL);
+        LogFatal() << "Cannot initialize runtime. The application name \"" << **name << "\" is not a valid file name.";
```

Comment:
I think we should give the user more info on what is allowed and what not. Where did you derive `isValidFileName` from? Is this defined in some POSIX / UNIX standard?

```suggestion
        LogFatal() << "Cannot initialize runtime. The application name \"" << **name << "\" is not a valid file name according to POSIX blublablubb";
```

Reply:
Please also update doxygen of `initRuntime` and add there how an valid name looks like e.g. `defined according to POSIX standard blublablubb`